src/components/Reports/FourteenMonthReports/TableHead/TableHead.tsx

```diff
@@ -12,7 +12,13 @@ import {
 import { FourteenMonthReportQuery } from '../GetFourteenMonthReport.generated';
 import { TableHeadCell } from './TableHeadCell/TableHeadCell';
 
+export type Unarray<T> = T extends Array<infer U> ? U : T;
+export type Contacts = FourteenMonthReportQuery['fourteenMonthReport']['currencyGroups'][0]['contacts'];
+export type Contact = Contacts[number];
+export type Months = Contact['months'];
+export type Month = Unarray<Months>;
```

Comment:
Is this the same as `export type Month = Months[0];`?

Can you use `ElementOf`? https://github.com/krzkaczor/ts-essentials#elementof-type I think it's the same and I like the name better. I've already brought that `ts-essentials` library in for something else.

We seem to need to do this a lot. I kinda forgot about these helpers. Might be useful elsewhere too. Thanks for the reminder :)

Reply:
Yeah, I will use the "essential type" rather use `Month[0]`.
But I guess it needs a little tricky since it has `Maybe` type for the `Months`.
Thank you. :)